space-provider/space-framework.md

```diff
@@ -0,0 +1,117 @@
+This document includes the text from the space-framework design document
+
+The full design document can be found here: https://docs.google.com/document/d/1WZCxlGoaCZVymqMhLCa5JZeURdCTzju8C5ABS8V4lW0/edit?usp=sharing
+
+
+# Space support for KubeStellar
+
+## Background
+
+KubeStellar architecture leverages spaces as the container building block for various purposes. For example, the separation between the KubeStellar service provider and the inventory is done by using different spaces for the inventory and the service provider. 
+
+The KubeStellar design currently 
+
+1. Separate different user workloads into different spaces (workload management spaces)
```

Comment:
Since this is a new document, I would prefer that we not create technical debt by using old terminology or create confusion by using mixed terminology.  We have a glossary now, see https://docs.kubestellar.io/main/Getting-Started/user-guide/#glossary .  If you mean to suggest a change in terminology please explicitly raise that point, as a distinct issue.

Reply:
Will verify

Reply:
@MikeSpreitzer reading the glossary, I am not clear on whether a workload descriptor space is the same as a space or is something different?